Sources/SnowManLogic/ViewLogic/View.swift

```diff
@@ -0,0 +1,64 @@
+public class View: Viewable {
+	var io: Receivable
+	var guess: String!
+
+	public init(io: Receivable) {
+		self.io = io
+	}
+
+	public func receiveGuess(getGuess: () -> String) -> String {
```

Comment:
I was trying to do something overly complicated. I removed it. I have to re-work my mocks.